airbyte-webapp/src/components/ui/Text/Text.tsx

```diff
@@ -45,9 +48,10 @@ export const Text: React.FC<React.PropsWithChildren<TextProps>> = React.memo(
     className: classNameProp,
     size = "md",
     inverseColor = false,
+    color = "default",
```

Comment:
I think we need to improve our Text component by accepting a `color` prop: https://github.com/airbytehq/airbyte/issues/22557

I'd follow up further refactoring in a separate PR